fineract-provider/dependencies.gradle

```diff
@@ -80,9 +80,10 @@ dependencies {
 
             'org.drizzle.jdbc:drizzle-jdbc',
 
-            'org.apache.poi:poi',
-            'org.apache.poi:poi-ooxml',
-            'org.apache.poi:poi-ooxml-schemas',
+            'org.apache.poi:poi:4.1.2',
```

Comment:
Since https://github.com/apache/fineract/pull/662, we don't want to set version numbers for prject dependencies in dependencies.gradle but in the build.gradle.
Version numbers don't really fit here anymore. Look at https://github.com/apache/fineract/pull/662 for more about this.

Reply:
OK - will have a look to see if there are duplicates, and fix that. Thanks!